Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -0,0 +1,511 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Team = Gordon360.Models.CCT.Team;
+using Gordon360.Models.CCT.Context;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Mail;
+using System.Globalization;
+using Microsoft.Graph;
+using Microsoft.AspNetCore.Server.IIS.Core;
+
+namespace Gordon360.Services.RecIM
+{
+    public class TeamService : ITeamService
+    {
+        private readonly CCTContext _context;
+        private readonly IMatchService _matchService;
+        private readonly IParticipantService _participantService;
+        private readonly IAccountService _accountService;
+        private readonly IConfiguration _config;
+
+        public TeamService(CCTContext context, IConfiguration config, IParticipantService participantSerivce, IMatchService matchService, IAccountService accountService)
+        {
+            _context = context;
+            _config = config;
+            _matchService = matchService;
+            _accountService = accountService;
+            _participantService = participantSerivce;
+        }
+        public IEnumerable<LookupViewModel> GetTeamLookup(string type)
+        {
+            if (type == "status")
+            {
+                var res = _context.TeamStatus
+                    .Select(s => new LookupViewModel
+                    {
+                        ID = s.ID,
+                        Description = s.Description
+                    })
+                    .AsEnumerable();
+                return res;
+            }
+            if (type == "role")
+            {
+                var res = _context.RoleType
+                    .Select(s => new LookupViewModel
+                    {
+                        ID = s.ID,
+                        Description = s.Description
+                    })
+                    .AsEnumerable();
+                return res;
+            }
+            return null;
+        }
+        public double GetTeamSportsmanshipScore(int teamID)
+        {
+            var sportsmanshipScores = _context.Match
+                                    .Where(m => m.StatusID == 6)
+                                        .Join(_context.MatchTeam
+                                            .Where(mt => mt.TeamID == teamID),
+                                            m => m.ID,
+                                            mt => mt.MatchID,
+                                            (m, mt) => new { score = mt.Sportsmanship }
+                                        );
```

Comment:
```suggestion
            var sportsmanshipScores = _context.MatchTeam
                                    .Where(mt => mt.Match.StatusID == 6 && mt.TeamID == teamID)
                                        .Select(mt => mt.Sportsmanship);
```

Reply:
6 could also use a comment as to what status that means